helm/alfresco-content-services/values.yaml

```diff
@@ -509,7 +509,7 @@ postgresql:
   # Note: Set this to false if you use an external database.
   enabled: true
   nameOverride: postgresql-acs
-  imageTag: "11.7.0"
+  imageTag: "13.1"
   commonAnnotations:
```

Comment:
Not sure this is correct, as I would not have expected 11.7.0 as the original value.